src/scripts/modal.js

```diff
@@ -0,0 +1,37 @@
+function modal() {
+  const heroContent = document.querySelector(".hero__content");
+  const modalContent = document.querySelector("#modal");
+  const checkOfferBtn = document.querySelector(".hero--cta");
+  const confrimModalBtn = modalContent.querySelector(".modal-btn--confirm");
+  const closeModalBtn = modalContent.querySelector(".modal-btn--close");
+  const closeModalIcon = modalContent.querySelector(".modal-icon--close");
+
+  function close() {
+    modalContent.classList.remove("modal--show");
+    heroContent.classList.remove("blur");
+
+    checkOfferBtn.disabled = false;
+
+    document.removeEventListener("click", closeOutside, true);
+  }
+
+  function open() {
+    modalContent.classList.add("modal--show");
+    heroContent.classList.add("blur");
+
+    checkOfferBtn.disabled = true;
+
+    document.addEventListener("click", closeOutside, true);
+  }
+
+  function closeOutside(e) {
```

Comment:
It's a good manner to define a function before using it. Please move it above `open` fn definition to improve readability

Reply:
Done